manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "trustpilot",
   "vendor": "vtex",
-  "version": "2.0.0",
+  "version": "2.1.0-beta.0",
```

Comment:
Updating the version here in the manifest is also handled by a bot. Please revert this back to `2.0.0`

Reply:
sorry! I thought @Danger was requesting me. i remove them